pkg/plugin/v2/plugin.go

```diff
@@ -61,3 +63,4 @@ func (p Plugin) GetCreateAPIPlugin() plugin.CreateAPI { return &p.createAPIPlugi
 
 // GetCreateWebhookPlugin will return the plugin for v2 which is responsible for scaffold webhooks for the project
 func (p Plugin) GetCreateWebhookPlugin() plugin.CreateWebhook { return &p.createWebhookPlugin }
+func (p Plugin) GetEditPlugin() plugin.Edit                   { return &p.editPlugin }
```

Comment:
@Adirio Needs the comments here as per linting policy. I am not sure why it is not throwing error for other places..